job_definitions/clean_and_migrate_db_dump.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: clean-and-migrate-db-dump
```

Comment:
I wonder if we need a different word for this. Right now we use migrate for both "schema migrations" and "moving data across environments", which is a bit confusing. Restore or apply would work to avoid the clash, but maybe we can find something more distinct.